diff --git a/port/linux/oc_config.h b/port/linux/oc_config.h
index 8e801156..7c8fbd78 100644
--- a/port/linux/oc_config.h
+++ b/port/linux/oc_config.h
@@ -34,7 +34,7 @@ typedef uint64_t oc_clock_time_t;
 /* Add support for the oic.if.create interface in Collections */
 #define OC_COLLECTIONS_IF_CREATE
 /* Add support for the maintenance resource */
-#define OC_MNT
+//#define OC_MNT
 /* If we selected support for dynamic memory allocation */
 #ifdef OC_DYNAMIC_ALLOCATION
 #define OC_COLLECTIONS
